apps/client/src/shared/apis/confeti-detail/concert.ts

```diff
@@ -0,0 +1,13 @@
+import { BaseResponse } from '@shared/types/api';
+import { ConcertDetailResponse } from '@shared/types/concert-response';
+import { END_POINT } from '@shared/constants/api';
+import { axiosInstance } from '../config/instance';
+
+export const getConcertDetail = async (
+  concertId: number,
+): Promise<ConcertDetailResponse> => {
+  const response = await axiosInstance.get<BaseResponse<ConcertDetailResponse>>(
+    `${END_POINT.GET_CONCERT_DETAIL}/${concertId}`,
+  );
+  return response.data.data;
```

Comment:
```suggestion
  const response = await get<BaseResponse<ConcertDetailResponse>>(
    `${END_POINT.GET_CONCERT_DETAIL}/${concertId}`,
  );
  return response.data.data;
```

axiosInstance내에 선언되어있는 get을 import해서 사용해주세요 !

Reply:
동의합니당~!